storage/innobase/fil/fil0fil.cc

```diff
@@ -821,6 +821,23 @@ class Fil_shard {
     mutex_release();
   }
 
+  void clear_ibts() {
+    mutex_acquire();
+
+    for (auto it : m_ibt_deleted) {
```

Comment:
Avoid copying, `Space` seems to be a pretty complicated structure
```suggestion
    for (const auto& it : m_ibt_deleted) {
```